sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -171,112 +182,184 @@ Sample code snippets are provided to illustrate using long-running operations [b
 
 The following section provides several code snippets covering some of the most common Form Recognizer tasks, including:
 
-* [Recognize Forms Using a Custom Model](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model")
-* [Recognize Content](#recognize-content "Recognize Content")
+* [Analyze Documents Using a Custom Model](#analyze-documents-using-a-custom-model "Analyze Documents Using a Custom Model")
+* [Extract layout](#extract-layout "Extract Layout")
 * [Using Prebuilt Models](#using-prebuilt-models "Using Prebuilt Models")
-* [Train a Model](#train-a-model "Train a model")
+* [Build a Model](#build-a-model "Build a model")
 * [Manage Your Models](#manage-your-models "Manage Your Models")
 
-### Recognize Forms Using a Custom Model
-Recognize name/value pairs and table data from forms. These models are trained with your own data, so they're tailored to your forms.
-For best results, you should only recognize forms of the same form type that the custom model was trained on.
+### Analyze Documents Using a Custom Model
+Analyze document fields, tables, selection marks, and more. These models are trained with your own data, so they're tailored to your documents.
+For best results, you should only analyze documents of the same document type that the custom model was built with.
 
 ```python
-from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import DocumentAnalysisClient
 from azure.core.credentials import AzureKeyCredential
 
-endpoint = "https://<region>.api.cognitive.microsoft.com/"
+endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
 credential = AzureKeyCredential("<api_key>")
 
-form_recognizer_client = FormRecognizerClient(endpoint, credential)
+document_analysis_client = DocumentAnalysisClient(endpoint, credential)
 model_id = "<your custom model id>"
```

Comment:
Nit?
```suggestion
model = "<your custom model id>"
```

Reply:
This model_id maps to the one from build so I think we should keep it.